examples/actions.py

```diff
@@ -1,54 +1,55 @@
 #!/usr/bin/env python3
 
-from xstate import Machine
 import time
 
+from xstate import Machine
+
 # Trafic light example
 # green -> yellow -> red -> green ..
 timing = 2
 
+
 def enterGreen():
     print("\tENTER_GREEN callback")
+
+
 def exitGreen():
     print("\tEXIT_GREEN callback")
 
+
 lights = Machine(
     {
         "id": "lights",
         "initial": "green",
         "states": {
             "green": {
-                "on": {"TIMER": "yellow"}, 
-                "entry": [{"type": "enterGreen"}], 
-                "exit": [{"type": "exitGreen"}]
-            },
-            "yellow": {
-                "on": {"TIMER": "red"}, 
-                "entry": [{"type": "enterYellow"}]
+                "on": {"TIMER": "yellow"},
+                "entry": [{"type": "enterGreen"}],
+                "exit": [{"type": "exitGreen"}],
             },
+            "yellow": {"on": {"TIMER": "red"}, "entry": [{"type": "enterYellow"}]},
```

Comment:
Is this the autoformatter's doing? Wonder if we can force newlines

Reply:
ah yes it is. With `black` you can do

```python
# fmt: off
<UNFORMATTED CODE>
# fmt: on
```

Probably good to do that in the examples